public/css/style.css

```diff
@@ -233,3 +233,12 @@ a {
     color: white;
     padding-bottom: 4px;
 }
+
```

Comment:
These have "+" signs in front of each line. Does js ignore those? Or do you need to delete them?